README.md

```diff
@@ -23,6 +23,13 @@ example), run:
 make SKIP_BUILD=true dev.run-on-kind
 ```
 
+Download the clusterclass quick start template:
+
+```shell
+wget https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.5.0/clusterclass-quick-start.yaml
+kubectl apply -f clusterclass-quick-start.yaml
+```
+
```

Comment:
The `clusterctl generate` command below actually creates this `ClusterClass` so unless you want to create it manually for other reasons this step is unnecessary.

```bash
$ clusterctl generate cluster capi-quickstart --flavor development --kubernetes-version v1.27.2 --control-plane-machine-count=1 --worker-machine-count=1 | gojq --yaml-input --yaml-output '. | select(.kind=="ClusterClass")'

apiVersion: cluster.x-k8s.io/v1beta1
kind: ClusterClass
metadata:
  name: quick-start
  namespace: default
<SNIP>
```

Reply:
Ah I see what the problem is, the `kubectl label` messes up the YAML. Tested the following and it works - I'll push a new commit with this change to this PR (can always revert if you prefer to keep it this way):

```bash
clusterctl generate cluster capi-quickstart \
    --flavor development \
    --kubernetes-version v1.27.2 \
    --control-plane-machine-count=1 \
    --worker-machine-count=1 | \
  gojq --yaml-input --yaml-output \
    '. | (select(.kind=="Cluster").metadata.labels["capiext.labs.d2iq.io/cni"]|="calico")' | \
  kubectl apply --server-side -f -
```

Reply:
On second thoughts, I'm going to merge #107 and could you test again please? I think that should have everything you need to get going.